index.html

```diff
@@ -40,201 +43,227 @@
+        </div>
+      </div>
     </section>
+      <!-- About section -->
+      <section class="About_Section">
+          <div class="About_Section_flex">
+          <h1>About <br />Myself</h1>
+          <p class="About_Section_description">
+            Hello I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+          <div class="links-text">Let's connect</div>
+          <ul class="icons-section">
+              <li>
+                  <a href="https://twitter.com/Lucas_David_22">
+                  <img src="./svg_files/twitter.svg" alt="twitter" /></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/lucas-erkana-b30a0b3b/">
+                <img src="./svg_files/Linkedin.svg" alt="Linkedin" /></a>
+              </li>
+              <li>
+                <a href="https://medium.com/">
+                <img src="./svg_files/medium.svg" alt="medium" /></a>
+            </li>
+            <li>
+              <a href="https://github.com/Lucash2022">
+                <img src="./svg_files/github.svg" alt="github" /></a>
+            </li>
+            <li>
+              <a href="https://www.angellist.com/">
+                <img src="./svg_files/angellist.svg" alt="angellist" />
+              </a>
+            </li>
+          </ul>
+          <div class="spacer"></div>
+          <button class="btn">Get my resume</button>
+          </div>
+          <div class="About_Section_flexx">
+              <div class="spacer"></div>
+              <ul class="about_section_lists">
+                <div class="dropdown">
+                  <li class="section_header">Languages</li><span>^</span>
+                </div>
+                    <ul class="languages">
+                        <li class="container">
+                          <img src="./svg_files/Js_ellipse.png" alt="js" />
+                          <div>Javascript</div>
+                        </li>
+                        <li class="container">
+                          <img src="./svg_files/Html_elipse.png" alt="html" />
+                          <div>Html</div>
+                        </li>
+                        <li class="container">
+                          <img src="./svg_files/Css_ellipse.png" alt="css" />
+                          <div>CSS</div>
+                        </li>
+                    </ul>
+                  <div class="dropdown">
+                    <li class="section_header">Frameworks</li><span>^</span>
+                  </div>
+                  <div class="dropdown">
+                    <li class="section_header">Skills</li><span>^</span> 
+                  </div>
+              </ul>
+          </div>
+      </section>
+<!-- footer -->
     <footer class="footer">
```

Comment:
[Fixed button](https://github.com/Lucash2022/Portfolio_Website/pull/4/commits/e606b9506937c0e67972dbdeedd1bdad0a52d9d6)

Reply:
[Fix background issue](https://github.com/Lucash2022/Portfolio_Website/pull/4/commits/836747e14127fa45676cb02ecd8fcb99ce6fca7f)

Reply:
[Fixed paragraph issue](https://github.com/Lucash2022/Portfolio_Website/pull/4/commits/6b339ceda1d90b421f325308d9131adf16aaf8a8)

Reply:
![image](https://user-images.githubusercontent.com/41912912/200100376-b67cd97a-ba39-447c-b709-cec56eb9e1e3.png)

Kindly, following the advice of the previous reviewer, I recommend adjusting this text to look more like the original Figma design, this will make your site look more professional

Reply:
@mgmediaweb 
[Adjusted the paragraph](https://github.com/Lucash2022/Portfolio_Website/pull/4/commits/cec9dc4a231bcae1e161a256dc3154d8025bd334)